module/VuFind/src/VuFind/DigitalContent/FakeOverdriveConnector.php

```diff
@@ -313,7 +313,7 @@ public function getAuthHeader()
     }
 
     /**
-     * Returns permanant links for Ovedrive resources
+     * Returns permanent links for Overdrive resources
      *
      * @param array $overDriveIds An array of overdrive IDs we need links for
```

Comment:
```suggestion
     * @param array $overDriveIds An array of OverDrive IDs we need links for
```

Reply:
Should this one also use the camel-case variant?

Reply:
Yes, good catch!